### DISCUSSION TITLE
Chip Template Card: Filter Entities

### DISCUSSION THREAD

**Discussion by teek541** [piitaya/lovelace-mushroom#1244] [Ideas]
It would be nice to be able to filter chips based on entities.  This could remove clutter for entities that are off/unavailable.

**Comment by piitaya** [comment]
I think you can do this with [auto entities](https://github.com/thomasloven/lovelace-auto-entities).

**Comment by teek541** [comment]
I was playing around with that yesterday.  Do you have a code example of this working with media player chips that I can just edit and run with?  I couldn't get it working.
